src/Response/DTO/Address.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\OverseasBundle\Response\DTO;
+
+class Address
+{
+    private ?string $addressId = null;
+    private string $name;
+    private string $zipCode;
+    private string $place;
+    private string $street;
+    private ?int $houseNumber = null;
+    private string $houseNumberAddition = '';
+    private string $countryPrefix;
+    private ?string $phone = null;
+    private ?string $fax = null;
+    private ?string $textPhone = null;
+    private ?string $email = null;
+    private ?float $lat = null;
```

Comment:
it'd be great to have a Coordinates VO like https://github.com/answear/econt-pickup-point-bundle/blob/master/src/Response/Struct/Coordinates.php

Reply:
I can do this by another method `getCoordinates`, but also cannot drop $lat and $long property because I have autodenormalization for this.
Check it out https://github.com/answear/overseas-bundle/pull/6